torch/nn/modules/activation.py

```diff
@@ -1304,10 +1304,11 @@ def __init__(self, dim: Optional[int] = None) -> None:
         self.dim = dim
 
     def __setstate__(self, state):
-        self.__dict__.update(state)
```

Comment:
@jbschlosser @albanD taking out this call since we call into super() now, and super() does `self.__dict__.update(state)` so this should be safe.

Reply:
Cool, sounds good :)